### PR TITLE
Add net.krafting.PedantiK

### PR DESCRIPTION
**Pull request by Krafting**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.

PedantiK is a game where you need to find a Wikipedia page (for now in french only)

A bit like Pedantle

  

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. 

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: net.krafting.PedantiK.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+app-id: net.krafting.PedantiK
+runtime: org.gnome.Platform
+runtime-version: '46'
```
Comment (hfiguiere) [inline net.krafting.PedantiK.yml:3]: 47 is out.

File: net.krafting.PedantiK.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    config-opts:
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    sources:
+      - type: git
+        url: https://github.com/OpenMathLib/OpenBLAS
+        tag: 'v0.3.26'
+        commit: 6c77e5e314474773a7749357b153caba4ec3817d
+
+  - name: LAPACK
```
Comment (hfiguiere) [inline net.krafting.PedantiK.yml:26]: does it really need lapack and blas for that?

File: net.krafting.PedantiK.yml (hunk 1 above)
Comment (Krafting) [reply net.krafting.PedantiK.yml:3]: True, I'll test it to see it it break anything

File: net.krafting.PedantiK.yml (hunk 2 above)
Comment (Krafting) [reply net.krafting.PedantiK.yml:26]: I had issue when one wasn't present I believe, I'll try again on my side.

Comment (hfiguiere) [reply net.krafting.PedantiK.yml:26]: I mean why do you need computational libraries for a game? (includes numpy).

Comment (Krafting) [reply net.krafting.PedantiK.yml:26]: gensim requires them both iirc (maybe I'm wrong, I'm re-testing all of that)

I use it to find words close to other words and such

Comment (hfiguiere) [reply net.krafting.PedantiK.yml:26]: ah ok.

Comment (Krafting) [reply net.krafting.PedantiK.yml]: Indeed, it does not seems to be required. It build just fine without it.

**Comment by Krafting** [comment]
bot, build net.krafting.PedantiK

**Comment by Krafting** [comment]
bot, build net.krafting.PedantiK

File: net.krafting.PedantiK.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        tag: 'v0.3.28'
+        commit: 5ef8b1964658f9cb6a6324a06f6a1a022609b0c5
+
+  - name: meson-python
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/mesonbuild/meson-python
+        tag: '0.15.0'
+        commit: e847bdd918ba6686d9b0502e9ee6d83d5145ae84
```
Comment (bbhtt) [inline net.krafting.PedantiK.yml:32]: this should be cleaned up with `- "*"` as cleanup

File: net.krafting.PedantiK.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      - install -Dm755 PedantiK.py /app/bin/PedantiK.py
+      - cp -r utils/ /app/bin/utils/
+      - install -Dm 0644 frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin  /app/models/frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin
+      - install -Dm 0644 net.krafting.PedantiK.desktop  /app/share/applications/net.krafting.PedantiK.desktop
+      - install -Dm 0644 net.krafting.PedantiK.metainfo.xml  /app/share/metainfo/net.krafting.PedantiK.metainfo.xml
+      - install -Dm 0644 images/net.krafting.PedantiK.svg /app/share/icons/hicolor/scalable/apps/net.krafting.PedantiK.svg
+    sources:
+      - type: file
+        url: https://embeddings.net/embeddings/frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin
+        sha256: d51f3259d1f6fe12155a447623a82695c6d72916b1b61e150e2715c034b6c880
```
Comment (bbhtt) [inline net.krafting.PedantiK.yml:60]: what is this file ?

Comment (Krafting) [reply net.krafting.PedantiK.yml:60]: This is a model file for word similarities, from here : https://fauconnier.github.io/#data

File: net.krafting.PedantiK.yml (hunk 3 above)
Comment (Krafting) [reply net.krafting.PedantiK.yml]: should be good

**Comment by Krafting** [comment]
bot, build net.krafting.PedantiK

**Comment by bbhtt** [comment]
Seems to be available in French only? Is this not localised to English yet ?

**Comment by bbhtt** [comment]
French only is fine although please add a note in the description tag saying that. Although I'd prefer if it had English localisation.

**Comment by Krafting** [comment]
> Seems to be available in French only? Is this not localised to English yet ?

For the first version, yes. Localisation will come after (I'm working on a better localisation workflow for my apps, and its clearly not ideal for now)

Also, there is a line in the description in the XML metadata file that say it is french only for now

```
    <p>Only French is supported for now.</p>
```

**Pull request closed without merging**